README_AWS.md

```diff
@@ -50,8 +50,28 @@ OSX:
 
 Test The Setup
 --------------
-1. cd openshift-ansible
-1. Try to list all instances:
+1. cd ost all instances
 ```
-  ./cloud.rb aws list
+  bin/cluster list aws ''
```

Comment:
Basically, the list playbook checks if cluster_id is an empty string, and if so it'll list all instances (which is a better test than asking the user to already have a defined cluster before running list as a test).  I'll add a comment to indicate that is what is being done.